### DISCUSSION TITLE
Any idea when Kovasap's vegeEmbed fix will make it into a release?

### DISCUSSION THREAD

**Discussion by gpierce3** [metasoarous/oz#188] [Q&A]
We ran into the webpack problem.
I've used his fork as a dependency and we're past it.
However, one of our tools does not like running into Git SHA dependencies when scanning deps.edn.
Doesn't affect the code running, it's more of an annoyance.

Still, I'd like to go back to pulling Oz from Clojars when the fix is incorporated.

**Comment by metasoarous** [comment]
Hi @gpierce3. Thanks for pointing this out.

I'd be happy to review a PR if you'd be so kind as to set one up. Which branch there have you been using? I didn't really see a lot of context about changes anywhere, so was wondering about the context.

Thanks again.

**Reply by gpierce3** [reply]
I guess I didn't provide much information, did I?

Basically, we pulled the latest version of Oz from clojars (2.0.0.alpha5) to use in our project.
Ran into the same problem Kovasap mentioned in PR #187.
I see that code was merged onto master on March 9th.

This isn't a new problem so I don't think I'll set up a PR.
Mainly I'm just curious when a new version of Oz containing the merged code will be posted on clojars.

Thanks for your quick reply, and my apologies for my tardy response.  I wasn't monitoring the email folder this showed up in.
